custom/icds_reports/migrations/sql_templates/database_functions/insert_into_daily_attendance.sql

```diff
@@ -25,7 +26,9 @@ BEGIN
     'form_location_lat, ' ||
     'form_location_long, ' ||
     'image_name ' ||
-    'FROM ' || quote_ident(_daily_attendance_tablename) || ' WHERE month = ' || quote_literal(_start_date) || ')';
+    'FROM ' || quote_ident(_daily_attendance_tablename) || ' ' ||
+    'WHERE month = ' || quote_literal(_start_date) || ' ' ||
+    'ORDER BY awc_id, submitted_on, inserted_at DESC)';
```

Comment:
is `submitted_on` a `date` or a `datetime` in this context? I guess it must be `date`?

Reply:
Yeah. it's defined as a date in the original UCR data source.